Phonebook.Frontend/src/app/shared/directives/feedback-drawer/feedback-drawer-sheet/feedback-drawer-sheet.component.html

```diff
@@ -1,8 +1,26 @@
 <mat-list>
-  <h3 mat-line i18n="FeedbackDrawerSheetComponent|Title of the Feedback Drawer@@FeedbackDrawerSheetComponentTitle">
-    Leave Feedback, report a Bug or submit a new Idea
+  <h3
+    mat-line
+    i18n="FeedbackDrawerSheetComponent|Title of the Feedback Drawer@@FeedbackDrawerSheetComponentTitle"
+    class="pb-text-align-center"
+  >
+    Leave Feedback, report a Bug or suggest a new Idea
   </h3>
-
+  <mat-list-item *ngIf="runtimeEnvironment.contactEmail">
+    <a
+      mat-button
+      color="primary"
+      target="_blank"
+      rel="noopener"
+      href="{{ runtimeEnvironment.contactEmail }}"
```

Comment:
There should be a "mailto". This will just redirect to the email like a html-link.